packit/distgit.py

```diff
@@ -737,3 +737,25 @@ def get_bugzilla_ids_from_changelog(changelog: str) -> list[str]:
         bugs = [bug.group(1) for bug in re.finditer(bz_regex, changelog, re.IGNORECASE)]
         logger.debug(f"Bug IDs found in the changelog: {bugs}")
         return bugs
+
+    def get_nvr(self, dist_git_branch):
+        """
+        Get the NVR for the branch.
+
+        Returns:
+            nvr
+        """
+        self.create_branch(
+            dist_git_branch,
+            base=f"remotes/origin/{dist_git_branch}",
+            setup_tracking=True,
+        )
+
+        self.update_branch(dist_git_branch)
+        self.switch_branch(dist_git_branch)
```

Comment:
I hope we don't need the repo afterwards, cause I definitely wouldn't have expected `get_*` to do this :D

Reply:
but the building happens after this, so to assure we are having the branch up to date for getting the NVR, I would say we need to do this

Reply:
Ok, fine by me.